firmware/Core/Inc/eps/eps_config.h

```diff
@@ -0,0 +1,10 @@
+#ifndef __EPS_CONFIG_H
+#define __EPS_CONFIG_H
+
+#include <stdint.h>
+
+extern uint32_t test_integer_config_eps; // const at compile time
```

Comment:
Please remove this file. Not needed. May conflict with a similarly-named file in the EPS dev branch.